src/organisms/subject-attend.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from 'react';
```

Comment:
```suggestion
"use client";

import { useEffect, useState } from 'react';
```
クライアントのコンポーネントなので, この `"use client"` ディレクティブが必要になります.